mapit_mysociety_org/templates/mapit/index.html

```diff
@@ -7,27 +8,190 @@
+                    <p>Make your campaigns more powerful, with targeted location-based messaging.</p>
+                    <h4>See it in action</h4>
+                    <a href="https://bettercare.prostatecanceruk.org/">
+                        <img src="{% static 'img/pcuk.jpg' %}" width="600" height="337" alt="">
+                    </a>
+                    <p>Prostate Cancer UK’s site shows <a href="https://bettercare.prostatecanceruk.org/">what care is available</a> in your own region – then encourages you to campaign for better.</p>
+                </div>
+                <div>
+                    <h3>Match your users to the right part of the web</h3>
+                    <p>When different users need sending to different places, MapIt can guide the way.</p>
+                    <h4>See it in action</h4>
+                    <a href="https://www.gov.uk/rubbish-collection-day">
+                        <img src="{% static 'img/govuk.jpg' %}" width="600" height="337" alt="">
+                    </a>
+                    <p>The UK Government uses MapIt to <a href="https://www.gov.uk/rubbish-collection-day">match users with their local council</a>, so they can check on services like recycling or allotment applications.</p>
+                </div>
+            </div>
+
         </div>
     </div>
 
+    <div class="homepage-section homepage-section--contrasting">
+        <div class="container">
+            <div class="homepage-cta">
+                <p>
+                    Want to know more before you sign up?<br>
+                    Read all about MapIt for non-developers.
+                </p>
+                <a href="/overview" class="btn btn--primary">Read more</a>
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
+
+{% block extra_js %}
+
+<script type="text/html" id="try-loading">
+    <div class="homepage-try-loading">
+        <div class="container">
+            <p>
+                <span class="loading-grid">
+                    <span></span> <span></span> <span></span>
+                    <span></span> <span></span> <span></span>
+                    <span></span> <span></span> <span></span>
+                </span>
+                Fetching boundaries…
+            </p>
+        </div>
+    </div>
+</script>
+
+<script type="text/html" id="try-error">
+    <div class="homepage-try-error">
+        <div class="container">
+            <p><%= error %></p>
+        </div>
+    </div>
+</script>
+
+<script type="text/html" id="try-result">
+    <div class="homepage-try-result">
+        <div class="homepage-try-result__map" id="homepage-leaflet"></div>
+        <div class="container">
+            <h2>Areas covering <%= postcode %></h2>
+            <div class="homepage-try-result__areas">
+              <% _.each(areas, function(area) { %>
+                <a class="homepage-try-result__area" href="/area/<%= area.id %>.html" data-areatype="<%= area.type %>" data-areaid="<%= area.id %>">
```

Comment:
Yes, agreed, I would like some way to go from an area box to its area page, and to get to a postcode page. Postcode is presumably easier to fix, it could have a "More details" link somewhere (next to the "Areas covering" header?).